src/cascadia/TerminalCore/TerminalDispatch.cpp

```diff
@@ -134,6 +134,69 @@ try
 }
 CATCH_LOG_RETURN_FALSE()
 
+bool TerminalDispatch::HorizontalTabSet() noexcept
+{
+    return true;
+}
+
+bool TerminalDispatch::ForwardTab(const size_t numTabs) noexcept
+{
+    const auto cursorPosition = _terminalApi.GetCursorPosition();
+    auto column = cursorPosition.X;
+    const auto row = cursorPosition.Y;
+    auto tabsPerformed = 0u;
+    const auto width = 80;

```

Comment:
I need a size which is provided by `_pConApi->GetConsoleScreenBufferInfoEx(csbiex)` in `adaptDispatch`. Wonder how to do the same in `_terminalApi.`

Reply:
meh, I'd just add a `til::point GetTerminalSize()` to `ITerminalApi` (or just make it return a `COORD` if the `til` type doesn't work in that header)